compiler/aarch64/codegen/UnaryEvaluator.cpp

```diff
@@ -116,6 +116,61 @@ TR::Register *OMR::ARM64::TreeEvaluator::lnegEvaluator(TR::Node *node, TR::CodeG
    return tempReg;
    }
 
+static TR::Register *inlineVectorUnaryOp(TR::Node *node, TR::CodeGenerator *cg, TR::InstOpCode::Mnemonic op)
+   {
+   TR::Node *firstChild = node->getFirstChild();
+   TR::Register *srcReg = NULL;
+
+   srcReg = cg->evaluate(firstChild);
+
+   TR::Register *resReg = cg->allocateRegister(TR_VRF);
```

Comment:
Please reuse `srcReg` as `resReg` when that is possible.
See `commonUnaryHelper()` in the same file.